tests/src/main/java/org/hibernate/beanvalidation/tck/tests/valueextraction/resolution/ValueExtractorResolutionAlgorithmTest.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Bean Validation TCK
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.beanvalidation.tck.tests.valueextraction.resolution;
+
+import javax.validation.ConstraintDeclarationException;
+import javax.validation.constraints.NotNull;
+
+import org.hibernate.beanvalidation.tck.beanvalidation.Sections;
+import org.hibernate.beanvalidation.tck.tests.AbstractTCKTest;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.jboss.test.audit.annotations.SpecAssertion;
+import org.jboss.test.audit.annotations.SpecVersion;
+import org.testng.annotations.Test;
+
+/**
+ * @author Guillaume Smet
+ */
+@SpecVersion(spec = "beanvalidation", version = "2.0.0")
+public class ValueExtractorResolutionAlgorithmTest extends AbstractTCKTest {
+
+	@Deployment
+	public static WebArchive createTestArchive() {
+		return webArchiveBuilder()
+				.withTestClass( ValueExtractorResolutionAlgorithmTest.class )
+				.build();
+	}
+
+	@Test(expectedExceptions = ConstraintDeclarationException.class)
```

Comment:
We have some tests for `ValueExtractorDefinitionException`. The `ValueExtractorDeclarationException` are still missing. I opened issues last week with all the missing parts.